main/lmdb/APKBUILD

```diff
@@ -2,7 +2,7 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=lmdb
 pkgver=0.9.24
-pkgrel=0
+pkgrel=1
```

Comment:
Why is this needed? (see 50e76a2dd93dff1d8d3b59616a822459fbfd08d7)

If the problem is that there is no `provides=pc:lmdb`, then why would a simple rebuild fix it?

Reply:
https://github.com/alpinelinux/aports/blob/db01a69b12d719e2c8ff643694499e38e5e9fb19/main/lmdb/APKBUILD#L41

Turns out this was the cause.